aiosmtpd/testing/helpers.py

```diff
@@ -65,9 +65,8 @@ def start(plugin):
         warnings.filterwarnings('always', category=ResourceWarning)
 
 
-def assert_auth_success(testcase: TestCase, code, response):
-    testcase.assertEqual(code, 235)
-    testcase.assertEqual(response, b"2.7.0 Authentication successful")
```

Comment:
Previous logic stops on mismatch of code. Hiding possibly important info contained in the message.

So I rewrite the comparo to directly compare the tuples; no more hidden info :-)